### ISSUE TITLE
babel as engine

### ISSUE THREAD

**Issue by jschoch** [arabold/serverless-test-plugin#3]
is this intended to be used with the babel engine?

**Comment by arabold** [comment]
The plugin will simply evaluate if your Lambda functions run without throwing any errors. Depending on your setup, the transformation of your code with Babelify and the optimizer plugin happens only during deployment, but not when running `sls function test`.

Having that said, generally this should work perfectly fine. It's just not testing the babelified but the original code.

**Comment by jschoch** [comment]
it appears to skip the functions with the engine set as babel in s-function

**Comment by arabold** [comment]
True. It checks explicitly for "`nodejs`" as part of the runtime name (`index.js`, line 154). Adding babel to the check should work though. You want to try?
